judge/views/problem.py

```diff
@@ -201,9 +201,69 @@ def get_context_data(self, **kwargs):
                                           context['description'], 'problem')
         context['meta_description'] = self.object.summary or metadata[0]
         context['og_image'] = self.object.og_image or metadata[1]
+
+        context['can_vote'] = self.object.can_vote(user)
+        # The vote this user has already cast on this problem.
```

Comment:
This should probably read more like `Get the vote this user has already cast on this problem.`